timescaledb/contribute-to-docs.md

```diff
@@ -80,19 +86,81 @@ or ending sentences with prepositions. Obeying these rules can often make
 language clearer but, in some cases, they make language more complicated. In
 that case, feel free to ignore them.
 
+## Headings
+
+All headings should be written in sentence case: capitalise only the first word
+in the heading, and proper nouns.
+
+For top-level page headings, and most section headings, use the simplest noun
+phrase possible. For example, instead of "Using hypertables", call the page
+"Hypertables".
+
+For level two sections that contain one or more procedures, use a simple verb
+phrase. For example, "Install TimescaleDB". For the associated procedures, use a
+level three heading, with the gerund form of the same heading. For example,
+"Installing TimescaleDB".
+
+## Processes and procedures
+
+We use processes and procedures to provide readers with numbered steps to achieve a specific goal.
+Processes contain procedures. If the task you are describing is very lengthy, or has a series of distinct components, break it up into a series of procedures within a process.
+
+Procedures contain these elements:
+
+1.  Level two section title in simple verb form
+1.  Short introduction
+
+1.  Open `<procedure>` tag
+
+1.  Level three procedure title in gerund verb form
+1.  Numbered steps
+1.  Screenshot
+
+2.  Close `</procedure>` tag
```

Comment:
@charislam Can i make markdownlint ignore this somehow?

Reply:
Just do this! `<!-- markdownlint-disable-next-line -->`
If that somehow breaks the numbering (though I don't think it should ...) you can also do this above the entire block:
`<!-- markdownlint-disable -->`
And reenable below: `<!-- markdownlint-enable -->`